### PR TITLE
Costume Size Setter Addon

### PR DESCRIPTION
**Pull request by TheColaber**
W_L told me to make this PR because I needed help with it...

This is an addon that I've been working on, but I need help because it has bugs... I listed them in the addon file...

If anyone could help me with these, that would be great :)

### CODE REVIEW

File: addons/costume-dimensions/addon.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Set costume dimensions",
+  "description": "Select a peice of a costume to set it's width and height.",
```
Comment (GrahamSH-LLK) [inline addons/costume-dimensions/addon.json:3]: Typo!

File: addons/costume-dimensions/userscript.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+//             inputEltX.remove();
+//             inputEltY.remove();
+//             labelX.remove();
+//             labelY.remove();
+//           }
+//           inputEltX = inputEltY = selected = null;
+//           break;
+//         }
+//         selected = e.detail.action.selectedItems[0].bounds;
+//         if (!inputEltX) {
```
Comment (GrahamSH-LLK) [inline addons/costume-dimensions/userscript.js:20]: Why. Not. A. /* */.

**Comment by TheColaber** [comment]
@GrahamSH-LLK and others, do not review this, this is in draft state.

**Comment by Explosion-Scratch** [comment]
I must say that's a lot of commented code you got there!

**Pull request closed without merging**